### PR TITLE
home task1

### PR DESCRIPTION
**Pull request by VadimChernetski**

### CODE REVIEW

File: vchernetski/month.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    	<BODY> 
+    		<table border="1" width ="300">
+          <caption>DECEMBER</caption>
+          <col width="14%"><col width="14%"><col width="14%"><col width="14%"><col width="14%"><col width="14%"><col width="14%"> <col width="14%">
+          <tr>
+            <th colspan="7">Январь</th>
+          </tr>
+          <tr>
+            <th>Пн</th>
+            <td align="left"> </td>
```
Comment (pluhin) [inline vchernetski/month.html]: Do not add align=left here, td does align to left by default, please remove it

File: vchernetski/python02.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят
+        вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+        Например:</p>
+        <p>#</p>
+        <p>Это комментарий</p>
+        <p>Комментарии можно вставлять в любое место в строке, но помните, что Python
+        проигнорирует всё, что идёт после <cite>'#'</cite>.</p>
+     <table border="0" width="1700">
```
Comment (pluhin) [inline vchernetski/python02.html]: here need 100%

File: vchernetski/python03.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      print '%d + %d = %d ' % (num1,num2,num1+num2)
+      print '%d * %d = %d ' % (num1,num2,num1*num2)
+      print '%d - %d = %d ' % (num1,num2,num1-num2)
+      print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)</pre>
+        <p>Как видите, здесь написано меньше кода — 8 строк вместо 12. Если нам нужно будет что-
+        либо поменять в функции, мы сможем сделать это, не создавая проблем для нашей программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p>
+       <table border="0" width="1700">
```
Comment (pluhin) [inline vchernetski/python03.html]: 100%

File: vchernetski/weather.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+          <tr>
+            <th>Город</th>
+            <th>Утро</th>
+            <th>День</th>
+            <th>Вечер</th>
+            <th>Ночь</th>
+          </tr>
+          <tr>
+            <th>Минск</th>
+            <td align="left">-5&#176;</td>
```
Comment (pluhin) [inline vchernetski/weather.html]: remove align

**Comment by VadimChernetski** [comment]
my homework haven't mistakes??

>

**Comment by pluhin** [comment]
Done, I've merged it

**Pull request merged**